doc/getting-started/overview_dev.rst

```diff
@@ -181,8 +179,8 @@ You must configure all of your target's BIOS settings before running the Board
 Inspector tool, because the tool records the current BIOS settings in the board
 configuration file.
 
-ACRN requires the BIOS settings listed in :ref:`gsg-board-setup` of the Getting
-Started Guide.
+ACRN requires the virtualization settings listed in :ref:`gsg-board-setup` of
```

Comment:
There are more than just virtualization settings in the BIOS that are needed, and I think we should mention BIOS here.
```suggestion
ACRN requires the BIOS settings listed in :ref:`gsg-board-setup` of
```